events/src/main/java/org/fao/geonet/events/user/GroupLeft.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * 
+ */
+package org.fao.geonet.events.user;
+
+import org.fao.geonet.domain.UserGroup;
+
+public class GroupLeft extends UserUpdated {
```

Comment:
1 sentence description for each added class would be nice.  IE what did the group leave?  (I know if I look at the code I can see it means that a user left a group but that is not clear from the class name)